src/io/iomanager.hpp

```diff
@@ -73,6 +73,14 @@ namespace dmxfish::io {
 				return control_desk_handle->find_column(set_id, column_id);
 			}
 
+			inline uint16_t get_global_illumination() {
+				if(control_desk_handle) {
```

Comment:
At initialization, the main fader is locked at position 0 and global illumination is initialized to zero. Hence it might make sense to also return 0 during initialization.

Reply:
Changed as of commit ae9e5b7e94c70c5f55f82b4c8a893f8405c655dd.

Reply:
I think it was before changing right: without x-touch you dont be able to change the value, so it should be initial high.
I thought the other way around: as long as you dont touch the fader the dimmer should be high.

Reply:
ok, then lets leave that for now